### PR TITLE
Add simply static methods for number/currency transformations

### PR DESCRIPTION
**Pull request by cristiangomeze**
This is just a wrapper for the getNumberTransformer and getCurrencyTransformer methods, which allow you to use a fluent interface like so:

```
NumberToWords::numberTransformer('en')->toWords(5120) // outputs "five thousand one hundred twenty"

NumberToWords::currencyTransformer('en')->toWords(5120, 'USD') // outputs "fifty dollars ninety-nine cents"
```

### CODE REVIEW

**Comment by kwn** [comment]
Hi @cristiangomeze 

Thank you for your contribution. I'd be more keen to provide simply static methods for number/currency transformations. E.g. 

```php
NumberToWords::transformNumber('en', 5120) // outputs "five thousand one hundred twenty"
NumberToWords::transformCurrency('en', 5120, 'USD') // outputs "fifty dollars ninety-nine cents"
```

Wouldn't that be even simpler?

**Review by cristiangomeze — COMMENTED** [review]
@kwn  done!

File: README.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ Then it can be used passing in numeric values to the `toWords()` method:
 ```php
 $numberTransformer->toWords(5120); // outputs "five thousand one hundred twenty"
 ```
+You can also use the fluent version:
+
+```php
+NumberToWords::numberTransformer('en')->toWords(5120) // outputs "five thousand one hundred twenty"
```
Comment (kwn) [inline README.md:45]: I think this requires an update according to the latest changes (static methods).

File: README.md [hunk 2]
```diff
@@ -59,6 +64,11 @@ Then it can be used passing in numeric values for amount and ISO 4217 currency i
 ```php
 $currencyTransformer->toWords(5099, 'USD'); // outputs "fifty dollars ninety nine cents"
 ```
+You can also use the fluent version:
+
+```php
+NumberToWords::currencyTransformer('en')->toWords(5099, 'USD') // outputs "fifty dollars ninety nine cents"
```
Comment (kwn) [inline README.md:70]: I think this requires an update according to the latest changes (static methods).

**Review by kwn — CHANGES_REQUESTED** [review]
Great job, thanks. 2 updates are required:
1) Fixing conflicts
2) Update of the readme file

Thanks in advance.

File: README.md (hunk 2 above)
Comment (cristiangomeze) [reply README.md]: Ups, my bad

**Comment by cristiangomeze** [comment]
I think it's ready 😅

**Review by kwn — APPROVED** [review]

**Comment by kwn** [comment]
Thanks @cristiangomeze 

Merged and released in 2.4.0. Thank you!

**Pull request merged**